### ISSUE TITLE
Beta-testing -- Adding realm to the dataloc class

### ISSUE THREAD

**Issue by jservonnat** [senesis/climaf#4]
When exploring the dataloc functionality with the example available in cmip5drs.py, I've had the case of getting two files for the following request:
```
urls_CMIP5_Ciclad=["/prodigfs/esg"]
dataloc(organization="CMIP5_DRS", url=urls_CMIP5_Ciclad)
cdef("frequency","monthly") ;  cdef("project","CMIP5")
tas1pc=ds(model="IPSL-CM5A-MR", experiment="historical", variable="pr", period="1860-1961")
files=tas1pc.selectFiles()
print files
```
Here is what I get:
```
/prodigfs/esg/CMIP5/merge/IPSL/IPSL-CM5A-MR/historical/mon/atmos/Amon/r1i1p1/v20111119/pr/pr_Amon_IPSL-CM5A-MR_historical_r1i1p1_185001-200512.nc /prodigfs/esg/CMIP5/merge/IPSL/IPSL-CM5A-MR/historical/mon/ocean/Omon/r1i1p1/v20111119/pr/pr_Omon_IPSL-CM5A-MR_historical_r1i1p1_185001-200512.nc
```
I've tried adding cdef("realm","atmos") but it didn't change the result.
```
cdef("frequency","monthly") ;  cdef("project","CMIP5") ; cdef("realm","atmos")
```
If you confirm that it is relevant then I'll try to do my first contribution to CliMAF by adding "realm" to dataloc.

**Comment by senesis** [comment]
Not considering the realm is actuallay an issue with CMIP5 data. I also discovered that recently. Actually, solving the ambiguity would mean to handle the "table' CMIP5 DRS attribute, which I find not really appealing :  I would like that CMIP5 DRS complexity does not impact on simpler cases, whcih implies some re-structuring on how flexible a dataset is defined. To be done, but maybe preferably by myself.

**Comment by jservonnat** [comment]
Sounds good to me ;).

**Comment by senesis** [comment]
Issue solved with commit 51aeeb4
